src/theme/Navbar/index.js

```diff
@@ -134,11 +133,11 @@ function Navbar() {
   }, [location]);
   const { isDarkTheme, setLightTheme, setDarkTheme } = useThemeContext();
   const { navbarRef, isNavbarVisible } = useHideableNavbar(hideOnScroll);
-  const { logoLink, logoLinkProps, logoImageUrl, logoAlt } = useLogo();
-  const onToggleChange = useCallback(
-    e => (e.target.checked ? setDarkTheme() : setLightTheme()),
-    [setLightTheme, setDarkTheme],
-  );
+
+  // const onToggleChange = useCallback(
+  //   e => (e.target.checked ? setDarkTheme() : setLightTheme()),
+  //   [setLightTheme, setDarkTheme],
+  // );
```

Comment:
Same, and you should have cleaned up the other code that used these values. I'll push.